DICTIONARIES.md

```diff
@@ -0,0 +1,19 @@
+# Dictionaries
```

Comment:
Not sure about the placement of this file. I would prefer something like `/docs/Dictionaries.md`. 

Also not sure whether in the API table of the component should be listed all values of the dictionary for better documentation and usage.

Reply:
I can move this file, I wasn't sure of the placement either. 

As for the API tables. The idea was, that we will generate variants from types/scss-settings/etc, so when we add a new variant we won't need to update all docs, just one place in the global dictionary. But you can change my mind :)